src/utils/beacon/BeaconPeers.tsx

```diff
@@ -52,64 +49,83 @@ export const BeaconPeers = () => {
     );
   }
 
-  return <PeersDisplay peerInfos={peersWithId} removePeer={removePeer} />;
+  return (
+    <Box>
+      {peersWithId.map(peerInfo => (
+        <PeersDisplay key={peerInfo.name} peerInfo={peerInfo} />
```

Comment:
if you have 2 connections with the same dApp it'll be a duplicate key error

Reply:
PeersDisplay is created only once for each peer, not for each connection.

Please check PeersDisplay implementation, where we iterate through peer's connection and create separate rows for each connected account

Reply:
as I understand each peer = each connection. I logged in with 2 different accounts to the same dApp. That's what I observe if I console.log(data):
<img width="571" alt="Screenshot 2024-03-07 at 14 03 32" src="https://github.com/trilitech/umami-v2/assets/129749432/0db9b5e9-ae25-434f-a3c6-938e7086f0fc">
<img width="1875" alt="Screenshot 2024-03-07 at 14 03 39" src="https://github.com/trilitech/umami-v2/assets/129749432/9d9269b2-ebd4-4142-9aa9-f18ed32dc9d2">

Reply:
It's ghostnet app + mainnet app (at least for me)
 
Different dAppIds & connections in slice, but the same name - hence the error with keys here

Let me fix that